code/modules/antagonists/gang/gang.dm

```diff
@@ -44,19 +42,19 @@
 	if we don't have a "handler" var, and a gang_handler exists, we need to grab it, since our "handler" is null.
 	if the gang exists, we need to join it; if the gang doesn't exist, we need to make it. */
 	var/found_gang = FALSE
-	if(!starter_gangster) // if they're a starter gangster according to the handler, we don't need to check this shit
-		for(var/datum/team/gang/G in GLOB.antagonist_teams)
-			if(G.my_gang_datum.handler) // if one of the gangs in the gang list has a handler, nab that
-				handler = G.my_gang_datum.handler
-			if(G.name == gang_name)
-				my_gang = G
-				found_gang = TRUE
-				break
+	for(var/datum/team/gang/G in GLOB.antagonist_teams)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names